examples/type_conversion.rs

```diff
@@ -0,0 +1,114 @@
+#[cfg(feature = "approx")]
+use {approx::assert_abs_diff_eq, ndarray::prelude::*, std::convert::TryFrom};
```

Comment:
```suggestion
#[cfg(feature = "approx")]
use std::convert::TryFrom;

#[cfg(feature = "approx")]
use approx::assert_abs_diff_eq;
#[cfg(feature = "approx")]
use ndarray::prelude::*;
```

I don't know if community standards are diverging. (I don't really like "use trees".) Seems like use trees require tool-assistance to write them correctly which I think is too much, it's not the Rust I like, smells like code golfing. Prelude imports should obviously not be joined in on the same line as anything else. Different crates should not be combined together in use trees. :slightly_smiling_face: 

The back and forth in PR review is a bit awkward so I'll just push in this fix.